package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -202,6 +202,24 @@ append_interface_name() {
 	xappend "--interface-name=$1,$2"
 }
 
+append_bus() {
+	local section="$1"
+	local option="$2"
+	local bus_option="$3"
+	local default="$4"
+	local _loctmp
+	[ -z "$default" ] && default="0"
+	config_get _loctmp "$section" "$option" "$default"
+	local value="$(get_bool "$_loctmp" "$_loctmp")"
+	if [ -n "$value" ]; then
+		case "$value" in
+			0) ;;
+			1) xappend "$bus_option" ;;
+			*) xappend "$bus_option=$value" ;;
+		esac
+	fi
+}
```

Comment:
Consider adding some empty lines to indicate logic steps/groups.

Reply:
Done, made more readable